webapp/README.md

```diff
@@ -0,0 +1,28 @@
+# Mattermost Web App
+
+This folder contains the client code for the Mattermost web app. It's broken up into multiple packages each of which either contains an area of the app (such as `playbooks` or `boards`) or shared logic used across other packages (such as the packages located in the `platform` directory). For anyone who's used to working in [the mattermost/mattermost-webapp repo](https://github.com/mattermost/mattermost-webapp), most of that is now located in `channels`.
+
+## npm Workspaces
+
+To interact with a workspace using npm, such as to add a dependency or run a script, use the `--workspace` (or `--workspaces`) flag. This can be done when using built-in npm commands such as `npm add` or when running scripts. Those commands should be run from this directory.
+
+```sh
+# Add a dependency to a single package
+npm add react --workspace=boards
+
+# Build multiple packages
+npm run build --workspace=packages/client --workspace=packages/components
+
+# Test all workspaces
+npm test --workspaces
+
+# Clean all workspaces that have a clean script defined
+npm run clean --workspaces --if-present
+```
+
+To install dependencies for a workspace, simply run `npm install` from this folder as you would do normally. Most packages' dependencies will be included in the root `node_modules`, and all packages' dependencies will appear in the `package-lock.json`. A `node_modules` will only be created inside a package if one of its dependencies conflicts with that of another package.
```

Comment:
Instead of writing `this` folder would it not be better to explicitly write `webapp` folder

Reply:
this is ok, because this readme in inside of `webapp` itself

Reply:
Yeah, I think "this folder" should work. I was thinking about that when I wrote thing line since I wanted to say "the webapp folder", but that feels a bit weird when this file is already in that folder